internal/transport/http2_server.go

```diff
@@ -293,7 +293,7 @@ func NewServerTransport(conn net.Conn, config *ServerConfig) (_ ServerTransport,
 		// closed immediately by the latter. Skipping the error here will help
```

Comment:
Are the comments still valid? Delete them?

Reply:
It was mostly still relevant, but not 100% accurate after the change.  Reworded.